lib/multilingual/locale_loader.rb

```diff
@@ -1,19 +1,17 @@
 # frozen_string_literal: true
 class ::Multilingual::LocaleLoader
-  include ::ApplicationHelper
+  attr_reader :controller
```

Comment:
Since an `attr_reader` is defined, it would be best to use it :slightly_smiling_face: (one of the main benefits is to allow easier refactoring)